pythonx/clap/extracted_fzy/.gitignore

```diff
@@ -0,0 +1,3 @@
+/target
+!.cargo
+*.lock
```

Comment:
I don't think the [faq](https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries) applies to our situation. Although it's a library, it's unpublished and actually used for vim-clap exclusively. No others depend on it, and we need to ensure people are always able to compile this Rust extension on their own. Keeping `Cargo.lock` is better for our case.